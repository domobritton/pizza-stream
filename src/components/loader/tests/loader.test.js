import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../';

describe('Loader', () => {
  test('snapshot-1', () => {
    expect(
      renderer
        .create(<Loader timer={60} />)
    ).toMatchSnapshot();
  });
  test('snapshot-2', () => {
    expect(
      renderer
        .create(<Loader timer={0} />)
    ).toMatchSnapshot();
  });
});
