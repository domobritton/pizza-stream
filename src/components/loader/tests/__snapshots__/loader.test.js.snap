// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader snapshot-1 1`] = `
<div
  className="css-11zhy3w"
>
  <img
    alt="loading"
    className="css-kmx4e7"
    src="loading.gif"
  />
</div>
`;

exports[`Loader snapshot-2 1`] = `
<div
  className="css-11zhy3w"
>
  <img
    alt="loaded"
    className="css-kmx4e7"
    src="loading.svg"
  />
</div>
`;
